### PR TITLE
package_numpy_1_9 update to explicitly specify ATLAS libs from toolshed

### PR DESCRIPTION
**Pull request by pjbriggs**
tl;dr:

This PR attempts to fix an issue with `package_numpy_1_9` and its dependency on `package_atlas_3_10`; essentially the former fails to use the ATLAS libraries provided by the latter when building `numpy`.

More details:

In cases where there is a pre-existing set of ATLAS libraries on the system, `package_numpy_1_9`  will compile against these and not the ones provided by `package_atlas_3_10`; this causes installation failures subsequently for other packages that depend on `package_numpy_1_9`. (This is what we're observing on our test server running Scientific Linux, when attempting to install e.g. `package_biopython_1_66`).

In cases where the system ATLAS is not picked up or doesn't exist, the installation failures may not be observed; however the toolshed ATLAS libs are not picked up either.

Essentially this is the same problem as that for `package_python_2_7_numpy_1_9` in PR #547; the fix implemented here is the same i.e. use the patched version of the `numpy` code from the Galaxy depot which explicitly includes the toolshed ATLAS.

### CODE REVIEW

**Comment by bgruening** [comment]
Thanks Peter!

**Comment by pjbriggs** [comment]
Thanks @bgruening - this appears to be working for me now.
